cql/ScreeningAverageRiskLibrary.cql

```diff
@@ -391,14 +394,21 @@ define RecommendationText:
             }
         else if Age21Through29 then
           { 'Cervical cancer screening using cervical cytology testing alone is next due on ' + ToString(ToDate(ProposedScreeningDate)) + '.' }
-        else
+        else if WillBeAge65OrLessAtNextScreening then
```

Comment:
It would help to add something to the L2 that describes this behavior.

Reply:
Added description re: [screening past age 65](https://mitre.sharepoint.com/:w:/r/sites/CervicalCancerCDS/Shared%20Documents/CDS%20Development/L2%20Artifacts/Working%20Documents/[CDC_CC_CDS_L2Development_v7.docx](https://mitre.sharepoint.com/:w:/r/sites/CervicalCancerCDS/Shared%20Documents/CDS%20Development/L2%20Artifacts/Working%20Documents/CDC_CC_CDS_L2Development_v7.docx?d=wc90d0228d64f420bb1565fb44a712630&csf=1&web=1&e=aQuDWh&nav=eyJjIjoxNjM2MTg1MzU3fQ)?d=wc90d0228d64f420bb1565fb44a712630&csf=1&web=1&e=aQuDWh&nav=eyJjIjoxNjM2MTg1MzU3fQ) to L2